packages/app/templates/ui-extensions/projects/checkout_ui/README.md

```diff
@@ -0,0 +1,87 @@
+
+> By default, the Checkout UI Extension is preconfigured as a [`Dynamic Extension Point`](https://shopify.dev/api/checkout-extensions/checkout#extension-points).
+> You can change it to your desired Extension Point location.
+
+
+To shape your extension you have the following collection of tools available:
+* [UI Components](https://shopify.dev/api/checkout-extensions/checkout/components)
+* [Shopify APIs](https://shopify.dev/api/checkout-extensions/checkout/extension-points/api).
+
+With these tools you'll get access to Shopify-enabled resources (such as [Analytics](https://shopify.dev/api/checkout-extensions/checkout/extension-points/api#analytics), [Localization](https://shopify.dev/api/checkout-extensions/checkout/extension-points/api#localization) or [Storage](https://shopify.dev/api/checkout-extensions/checkout/extension-points/api#storage)) and the frameworks that you need to read (and mutate) data for your Checkout-UI extension (like Shop, Customer, CartLine, MetaFields, etc.).
+
+> If you are using React, there is also a large collection of [React Hooks available](https://shopify.dev/api/checkout-extensions/checkout/extension-points/api#react-hooks) to ease access to these operations, otherwise you'll need to manually subscribe to the subscribable value directly with a callback.
+
+## FAQ
+---
+* **How can I preview my extension**
+
+    Depending on your extension location they may be either `dynamic` or `static` renderers, which diferr slightly on the process to place them for testing.
+    - `Dynamic renderers`: Using a [query string parameter](https://shopify.dev/apps/checkout/test-ui-extensions#dynamic-extension-points) at the end of the URL you can place your renderer in the desired renderer point.
+    - `Static renderer`: No special work is required here, just note that a static renders is shown only if the section that they are attached to is enabled.
+
+* **How can I change my extension renderer type?**
+
+    To change your extension render type, be mindful that you have to change it in the following places:
+    1. In your `script file`, where you declared the extension (will be either _render()_ or _extend()_), you'll have to change the declared Extension Point.
+    2. In your `settings TOML file` (shopify.ui.extension.toml) you'll have to change the `extension_points` declaration for your new desired type.
+
+## Useful Links
+---
+
+- [Checkout app documentation](https://shopify.dev/apps/checkout)
+
+- [Checkout UI extension documentation](https://shopify.dev/api/checkout-extensions)
+  - [Configuration](https://shopify.dev/api/checkout-extensions/checkout/configuration)
+  - [API Reference](https://shopify.dev/api/checkout-extensions/checkout/extension-points/api)
+  - [UI Components](https://shopify.dev/api/checkout-extensions/checkout/components)
+  - [Available React Hooks](https://shopify.dev/api/checkout-extensions/checkout/extension-points/api#react-hooks)
```

Comment:
I think we can remove this, it's only relevant for developers using React and I think we are going to remove a bunch of these soon

Reply:
I'd still keep it here meanwhile they exist, as easy access for react developers, in this final section of the readme. See this as a utilitarian link